base/linalg/bidiag.jl

```diff
@@ -67,6 +67,10 @@ function convert{T}(::Type{Tridiagonal{T}}, A::Bidiagonal)
 end
 promote_rule{T,S}(::Type{Tridiagonal{T}}, ::Type{Bidiagonal{S}})=Tridiagonal{promote_type(T,S)}
 
+# Convert from Bidiagonal{Told} to Bidiagonal{Tnew} or AbstractMatrix{Tnew}(->Bidiagonal{Tnew})
+convert{Tnew,Told}(::Type{Bidiagonal{Tnew}}, A::Bidiagonal{Told}) = Bidiagonal(convert(Vector{Tnew}, A.dv), convert(Vector{Tnew}, A.ev), A.isupper)
+convert{Tnew,Told}(::Type{AbstractMatrix{Tnew}}, A::Bidiagonal{Told}) = convert(Bidiagonal{Tnew}, A)
```

Comment:
Shall I just nix the comment altogether? Those lines seem more or less self-explanatory. Or do you prefer the comment? If the latter, I'll break it in two. Thanks!